laboratorium1/przykładowy.c

```diff
@@ -0,0 +1,34 @@
+#include <stdio.h>
+
+int main () {
+float a;
+float b;
+float c;
+scanf("%f", &a);
+printf("%f\n", a);
+scanf ("%f", &b);
+printf("%f\n", b);
+scanf ("%f", &c);
+printf("%f\n", c);
+float wynik = (a*b) - (c*a) - (b*c);
+printf ("%f\n", wynik);
+return 0;
+int a,b,c;
+if ((a>b) && (c>a) && (c>b)) {
```

Comment:
Jeżeli **c** jest wieksze od **a** oraz **a** jest większe od **b**, to wiadomo, że **c** jest większe od **b**.
Ostatni warunek jest niepotrzebny.